### PR TITLE
Fix missing consts in CLI args

### PR DESCRIPTION
**Pull request by disconnect3d**
Brings back config variables into CLI arguments.

Also removed duplicated arguments.

This change is [Reviewable](https://reviewable.io/reviews/trailofbits/manticore/1337)

### CODE REVIEW

File: manticore/core/manticore.py [hunk 1]
```diff
@@ -39,16 +43,19 @@ class ManticoreBase(Eventful):
 
     _published_events = {'start_run', 'finish_run', 'generate_testcase'}
 
-    def __init__(self, initial_state, workspace_url=None, policy='random', **kwargs):
+    def __init__(self, initial_state, workspace_url=None, policy='random', timeout=None, **kwargs):
```
Comment (feliam) [inline manticore/core/manticore.py]: Do we need the timeout here if it is already a global config value ?

Comment (disconnect3d) [reply manticore/core/manticore.py]: It's the question where should the user specify it if they want to do so. Do you think they should use `config.core.timeout` instead?

Comment (disconnect3d) [reply manticore/core/manticore.py]: Let's leave it as an argument here as someone might want to use it when playing with scripts.

Comment (feliam) [reply manticore/core/manticore.py]: Ok. Note though that she may still need to do some config.py setting even from a script.

File: manticore/core/manticore.py [hunk 2]
```diff
@@ -414,7 +421,7 @@ def _finish_run(self, profiling=False):
 
         self._publish('did_finish_run')
 
-    def run(self, procs=1, timeout=0, should_profile=False):
+    def run(self, procs=1, should_profile=False, timeout=None):
```
Comment (disconnect3d) [inline manticore/core/manticore.py]:
```suggestion
    def run(self, procs=1, timeout=None, should_profile=False):
```

**Pull request merged**